ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/DdlDmlResponseHandlers.java

```diff
@@ -37,6 +37,10 @@ static void handleExecuteStatementResponse(
       final JsonObject ksqlEntity,
       final CompletableFuture<ExecuteStatementResult> cf
   ) {
+    if (isIfNotExistsWarning(ksqlEntity)) {
+      cf.complete(new ExecuteStatementResultImpl(Optional.empty()));
```

Comment:
Let's return the warning message instead of just empty.

Reply:
sounds good

Reply:
Hm yeah that's weird too. One way around this would be to add a warning field to `ExecuteStatementResult`. Otherwise, leaving the QueryID blank is what we do for `DROP` statements, so perhaps it's not the worst thing to leave it blank.

Reply:
I think we should leave it blank. For the UI and a CLI a warning will print still. This is only for the java client and that might as well just check the logs.